gui/scripts/syncthing/core/controllers/syncthingController.js

```diff
@@ -368,7 +367,16 @@ angular.module('syncthing.core')
                     id;
 
                 prevDate = now;
-                $scope.connections_total = data['total'];
+
+                try {
+                    data.total.inbps = Math.max(0, (data.total.inBytesTotal - $scope.connections_total.inBytesTotal) / td);
+                    data.total.outbps = Math.max(0, (data.total.outBytesTotal - $scope.connections_total.outBytesTotal) / td);
+                } catch (e) {
+                    data.total.inbps = 0;
+                    data.total.outbps = 0;
+                }
+                $scope.connections_total = data.total;
```

Comment:
Make this camelCase while we're anyway digging around here, as everything else is. Otherwise LGTM. :)